extensions/amazon-sales/release-notes/index.md

```diff
@@ -9,17 +9,20 @@ See the following documentation:
 
 - [Amazon Sales Channel](https://docs.magento.com/m2/ce/user_guide/sales-channels/amazon/amazon-sales-channel.html) for merchant information and instructions
 - [Amazon Sales Channel install]({{site.baseurl}}/extensions/amazon-sales/) for installation and API key information
-- [Amazon Sales Channel Marketplace page](http://marketplace.magento.com/magento-module-amazon.html)
+- Amazon Sales Channel Marketplace page (coming soon) 
+<!--- (http://marketplace.magento.com/magento-module-amazon.html)
+-->
 
 The release notes include:
 
 -   {:.new}New features
 -   {:.fix}Fixes and improvements
 -   {:.bug}Known issues
 
-### v1.0.0
+### v2.0.0
 
-Amazon Sales Channel 1.0.0 is compatible with version 2.2.4+ and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
+Amazon Sales Channel 2.0.0 is compatible with version 2.2.4+ and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
+**Note** Version 1.0 was available in limited release only. 
```

Comment:
This note may need to be in a note format. 
```suggestion

{:.bs-callout .bs-callout-info}
Version 1.0.0 was available in limited release only. 
```

Reply:
Good catch Lori. We'll get that updated.